content/overview/workflow/edit-overview.md

```diff
@@ -10,33 +10,39 @@ This section will guide you through the basics of developing an image in the [da
 
 To begin, open an image in the darkroom by double clicking its thumbnail in the [lighttable](../../lighttable/_index.md) view. 
 
-Each change you make to the image in the darkroom is turned into a history stack item. The history is stored in a database and in an XMP sidecar file for that image. All changes are automatically stored when you switch images or go from one darktable view to another. You can safely leave the darkroom or quit darktable at any time and come back later to continue your work. For this reason darktable does not need a “save” button and it does not have one.
+In the darkroom mode, you see a list of modules to the right hand side of the screen. Each module performs one processing step to bring an image from raw format into an image that can be seen on the screen or printed. The processing steps are performed in the order shown in the module list, starting at the bottom of the list and moveing up to the top, like a stack of building blocks where each block builds on the processing done by the modules below. The order in which the modules are applied is important to provide a correctly processed image.
 
-On the left panel of the darkroom view is the [history stack](../../module-reference/utility-modules/darkroom/history-stack.md) module, which lists the changes you have made, starting from the bottom -- each edit adds a new item to the top of the stack. You can select an earlier point in the history to show how the image looked at that point. The stack can be compressed to remove redundant intermediate points in your edits -- when you are happy with what you have done, just compress the history stack. Note that all edits above the selected history entry are permanently deleted when compressing the history stack.
+On the right-hand side, there is a history stack, which shows the order in which adjustments were made the the controls of the various modules. This allows you to undo changes by going back to an earlier step in the history stack.
 
-A large number of [processing modules](../../module-reference/processing-modules/_index.md) are shipped with darktable, arranged into [module groups](../../darkroom/organization/module-groups.md). These groups are accessed via buttons at the top of the right-hand panel, just below the [histogram](../../module-reference/utility-modules/shared/histogram.md).
+# typical editing workflow
 
-# choosing a workflow
-
-When processing an image, a sequence of processing modules is applied. This sequence is known as the [pixelpipe](../../darkroom/pixelpipe/_index.md). 
+When processing an image, the order in which you make adjustments to modules is not necessarily the same as the order in which the modules appear in the pixel pipe. The following diagram shows key modulesin the pixel pipe that are typically involved in basic processing of an image, and underneath are numbers showing the order in which you would normally adjust those modules.
```

Comment:
modules in

Reply:
It's an awkward sentence; I tried to rephrase.

Reply:
@matt-maguire  you have a typo:

diagram shows key **modulesin** the pixel pipe

Reply:
that typo was removed when I reworded the sentence.